stages/org.osbuild.modprobe

```diff
@@ -38,7 +43,7 @@ SCHEMA = r"""
           "additionalProperties": false,
           "type": "object",
           "description": "'blacklist' command",
-          "required": ["command","modulename"],
```

Comment:
Snuck in a whitespace change! Good that David is not here! ;)

Reply:
Oh yeah 🙄  In my defense I had good intentions 😄